### DISCUSSION TITLE
ForeignKey is reverse LinkToAnotheRecord [Aug 30]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3733] [Discord Threads]
> 🙋‍♂️ **azriel46d**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### You can use LinkToAnotherRecord which is reverse of ForeignKey  
#  
> 💁 **o1lab**  
#### nested querying is possible  
#  
> 💁 **o1lab**  
#### see our docs  
#  
> 💁 **o1lab**  
#### you can click on your project API docs itself, it will be easier  
#  
> 💁 **o1lab**  
#### 😄  
#  
> 💁 **o1lab**  
#### bottom left there is API docs and in top right menu there is swagger docs  
#  
> 🙋‍♂️ **azriel46d**  
#### <@310157751395287040> I am using as described above the LinkToAnotherRecord. Due to the need of having multiple records poiniting towards I end up having to utilise the many to many approach.  So over the API i get this   
#### ```   
#### {   
####       "Id": 4,   
####       "value": true,   
####       "feature": [   
####         {   
####           "Id": 4,   
####           "name": "xxx.yyy.zzz"   
####         }   
####       ]   
####     },   
#### ```   
#### I have tried to utilise in filtering to be able to get it to.    
#### ```   
#### {   
####   "value": true,   
####   "feature": "xxx.yyy.zzz"   
#### }```   
#### however utilising the docs and trying different strategies to try to obtain the first item in the array, does not make this happen. Is this the case? Or am i missing something ?  
#  
> 🙋‍♂️ **James2169**  
#### Why has ForeignKey field type been removed?  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1009111557311045652)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
